clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -277,6 +284,18 @@ function registerCommands(
         ),
     ),
   );
+
+  context.subscriptions.push(
+    vscode.commands.registerCommand("cobol-lsp.open.copybook.folder", () => {
```

Comment:
It is a bit confusing. We may have multiple copybook folders, why do we open only extension one?

Reply:
I think the idea was to make the internal folder available, to compensate for the `.c4z` folder not being in the workspace anymore.

Reply:
Righ, but the user can specify any folders in or outside the working folder, and can even use wildcards.

Reply:
Yes, but since those are actually provided by the user, the user knows where to find them, unlike the internal folder which is provided by vscode/extension.

Reply:
Should it be named something like "internal copybook folder" then?

Reply:
resolved at : 4ee00c309783183fda8cadd1a320b908e9379425

Reply:
> resolved at : [4ee00c3](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/commit/4ee00c309783183fda8cadd1a320b908e9379425)

What about the command id?

Reply:
resolved at : 05a16e62245fc7d6fe39f69736d3c431dc50236d